### PR TITLE
docs(theme): updated related docs with adding font family

### PR DESCRIPTION
**Pull request by ozkalai**
In this PR, We have updated with information about how to customize and apply font family in your Refine application in the `Customization Theme` and `Tutorial` documentation of Material UI.

### CODE REVIEW

**Review by omeraplak — APPROVED** [review]

**Review by biskuvit — APPROVED** [review]

File: documentation/docs/ui-frameworks/mui/tutorial.md [hunk 1]
```diff
@@ -236,6 +238,31 @@ export default App;
 `<GlobalStyles>` is a component that is used to apply the global styles to the application. You may refer to the [**GlobalStyles** documentation](https://mui.com/material-ui/api/global-styles/) for more information.
 :::
 
+:::info
+Font family comes in the theme as specified in the [`typography`](https://github.com/pankod/refine/blob/next/packages/mui/src/theme/typography.ts) property. Refine application uses [`Montserrat`](https://fonts.google.com/specimen/Montserrat?query=mont) font as default. You have to embed to `index.html` if you want to use [`Montserrat`](https://fonts.google.com/specimen/Montserrat?query=mont) as font family.
```
Comment (omeraplak) [inline documentation/docs/ui-frameworks/mui/tutorial.md:242]:
```suggestion
Font family comes in the theme as specified in the [`typography`](https://github.com/pankod/refine/blob/next/packages/mui/src/theme/typography.ts) property. Refine application uses [`Montserrat`](https://fonts.google.com/specimen/Montserrat?query=mont) font as default. You have to embed to `index.html` if you want to use [`Montserrat`](https://fonts.google.com/specimen/Montserrat?query=mont) as the font family.
```

File: documentation/docs/ui-frameworks/mui/tutorial.md [hunk 2]
```diff
@@ -236,6 +238,31 @@ export default App;
...
+For more information about to add font family in your Refine application, you can look at [`Material UI Theme Customization`](/ui-frameworks/mui/customization/theme.md#overriding-variables).
```
Comment (omeraplak) [inline documentation/docs/ui-frameworks/mui/tutorial.md:243]:
```suggestion
For more information about adding font family in your Refine application, you can look at [`Material UI Theme Customization`](/ui-frameworks/mui/customization/theme.md#overriding-variables).
```

File: documentation/docs/ui-frameworks/mui/tutorial.md [hunk 3]
```diff
@@ -236,6 +238,31 @@ export default App;
...
+    <body>
+        ...
+    </body>
+</html>
+```
+
+:::
+
 :::tip
 **refine** comes natively with Light/Dark theme support. Check out the [theme](#) documentation for more information.
```
[the remark this answers is not in the document]
Comment (aliemir) [reply documentation/docs/ui-frameworks/mui/tutorial.md:267]: And we have a missing link here 🤔

File: documentation/docs/ui-frameworks/mui/tutorial.md (hunk 1 above)
Comment (aliemir) [inline documentation/docs/ui-frameworks/mui/tutorial.md:242]:
```suggestion
Refine application uses [`Montserrat`](https://fonts.google.com/specimen/Montserrat) font by default as it is defined in the [`typography`](https://github.com/pankod/refine/blob/next/packages/mui/src/theme/typography.ts) property of the theme.  But to use [`Montserrat`](https://fonts.google.com/specimen/Montserrat), you need to embed it to your `index.html` file.
```

**Pull request merged**